tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -748,6 +759,134 @@ def only_ids(trades):
     assert len(all_offers) == 2
 
 
+@pytest.mark.asyncio
+async def test_did_endpoints(wallet_rpc_environment: WalletRpcTestEnvironment):
+    env: WalletRpcTestEnvironment = wallet_rpc_environment
+
+    wallet_2: Wallet = env.wallet_2.wallet
+    wallet_1_node: WalletNode = env.wallet_1.node
+    wallet_2_node: WalletNode = env.wallet_2.node
+    wallet_1_rpc: WalletRpcClient = env.wallet_1.rpc_client
+    full_node_api: FullNodeSimulator = env.full_node.api
+
+    await generate_funds(env.full_node.api, env.wallet_1, 5)
+
+    # Create a DID wallet
+    res = await wallet_1_rpc.create_new_did_wallet(1)
+    assert res["success"]
+    did_wallet_id_0 = res["wallet_id"]
+    did_id_0 = res["my_did"]
+    # Check DID ID
+    res = await wallet_1_rpc.get_did_id(did_wallet_id_0)
+    assert res["success"]
+    assert did_id_0 == res["my_did"]
+    # Create backup file
+    res = await wallet_1_rpc.create_did_backup_file(did_wallet_id_0, "backup.did")
+    assert res["success"]
+
+    for _ in range(3):
```

Comment:
We are not waiting for the TX to enter the mempool before farming the blocks. This means this test will be flaky, and is not guaranteed to succed. Also, if you check for mempool inclusion, you only need to farm one block not 3